doc/manuals/installation_and_administration_manual/streaming/annex_c_deploy_storm_cluster.md

```diff
@@ -0,0 +1,191 @@
+#<a name="top"></a>Annex C: Deploy Apache Storm Cluster
+
+Content:<br>
+
+* [Prepare Operating System](#section1)
+* [Deploy Storm](#section2)
+
+In order to complete Sinfonier installation it's necessary to deploy an Apache Storm cluster. This reference is based on CentOS 7 operating system and some specific software versions.
+
+##<a name="section1"></a>Prepare Operating System
+
+This guide use [CentOS 7](https://www.centos.org/) as based system where deploy Storm.
+
+**Add repositories**
+
+    https://github.com/CentOS/sig-atomic-buildscripts
+
+**Update your system**
+
+    $ sudo yum update
+
+**Install Java 7 SDK**
+
+    http://www.oracle.com/technetwork/es/java/javase/downloads/jdk7-downloads-1880260.html
+
+**Install Development tools**
+    
+    $ sudo yum groupinstall "Development Tools"
+
+[Top](#top)
+
+##<a name="section2"></a>Deploy Storm
+
+### Install zookeeper
+
+**Add cloudera repositories**
+
+To download and install the CDH4 "1-click Install" package
+
+    $ wget http://www.cloudera.com/content/www/en-us/documentation/archive/cdh/4-x/4-7-1/CDH4-Installation-Guide/cdh4ig_topic_4_4.html
```

Comment:
I would delete the blank lines between commands in order the 3 commands are rendered in the same text box.